resources/views/login.blade.php

```diff
@@ -1,43 +1,93 @@
-
-@extends('layout/layout') 
+<!--
+///////////////////////////////////////
+//
+// Jorge Pereda - 2021/03/10
+//  * ...
+//  * ...
+//
+///////////////////////////////////////
+-->
+@extends('layout/layout')
 @section('content')
 
+    <div class="flex max-w-sm mx-auto overflow-hidden bg-white rounded-lg shadow-lg dark:bg-gray-800 lg:max-w-4xl">
+        <div class="hidden bg-cover lg:block lg:w-1/2" style="background-image:url('https://images.unsplash.com/photo-1606660265514-358ebbadc80d?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1575&q=80')"></div>
 
-<h4 class="text-center mt-20 text-white font-inter text-5xl">Login</h4>
-<form class="text-white m-auto" action="{{ route('check') }}" method="post">
-@if(Session::get('fail'))
-    <div class="">
-        {{ Session::get('fail') }}
-@endif
-@csrf
-    <div class="text-center mt-20">
-        <h1 class="text-2xl mb-1">Email</h1>
-        <input type="text" class="w-96 h-9 p-2 rounded-lg text-black" name="email" value="{{ old('email') }}" placeholder="Your Email">
-        
-    </div>
-    <div class="text-center text-red-500">
-        <span>@error('email'){{ $message }} @enderror</span>
-    </div>
-    
-    <div class="text-center mt-5">
-        <h1 class="text-2xl mb-1">Password</h1>
-        <input type="password" class="w-96 h-9 p-2 rounded-lg text-black" name="password" placeholder="********">
-       
-    </div>
-    <div class="text-center text-red-500">
-        <span>@error('password'){{ $message }} @enderror</span>
-    </div>
-   
-    <div class="text-center mt-4">
-        <button type="submit">Sign In</button>
-    </div>
- 
+        <div class="w-full px-6 py-8 md:px-8 lg:w-1/2">
+            <h2 class="text-2xl font-semibold text-center text-gray-700 dark:text-white">IMDB</h2>
+
+            <p class="text-xl text-center text-gray-600 dark:text-gray-200">Group 05</p>
+
+            {{-- <a href="#" class="flex items-center justify-center mt-4 text-gray-600 rounded-lg shadow-md dark:bg-gray-700 dark:text-gray-200 hover:bg-gray-100 dark:hover:bg-gray-600">
```

Comment:
Should this be commented or removed?

Reply:
Om den är till google ska den vara kommenterad eftersom jag tänker lägga till ett registrerings system genom google om tiden finns.